attrs_strict/_commons.py

```diff
@@ -0,0 +1,15 @@
+def _is_newtype(type_):
+    """Checks if `type_` is :func:`typing.NewType`."""
+    return (
+        hasattr(type_, "__name__")
+        and type_.__module__ == "typing"
+        and type_.__name__ == "SomeNew"
+    )
+
+
+def _format_type(type_):
```

Comment:
Since this is being imported from `_commons` into `_errors`, it is not a private function anymore so the convention is to not use `_` as the start of a function name. Can you please rename it accordingly ?  Same applies to `_is_newtype` -> `is_newtype`.

Reply:
correct is, done.